src/v8/replace-node-with-v22-in-gitlab-ci-files.ts

```diff
@@ -0,0 +1,25 @@
+import { promises as fs } from "fs";
+import * as glob from "glob";
+
+export const stage = "before-install";
+
+/**
+ * Replace node with version 22 in .gitlab-ci yml files
+ */
+export default async function ReplaceNodeWithV22InNvmrc() {
+    const files = glob.sync("**/.gitlab-ci/*.yml", { nodir: true });
+
+    for (const file of files) {
+        let content = await fs.readFile(file, "utf8");
+
+        content = content
+            .replace(/node18/g, "node22")
+            .replace(/node20/g, "node22")
+            .replace(/nodejs18/g, "nodejs22")
+            .replace(/nodejs20/g, "nodejs22");
```

Comment:
I'm not sure if this is too primitive. But it works at least in our newer projects

Reply:
I recently had to update the Admin server because of changing to the minimal images (I didn't know that "normal" images still exist). This wouldn't be covered by this script (which is fine).

@dkarnutsch @kaufmo what do you think about this?

Reply:
I think this is ok. The regular images still exist. The change was probably due to envsubst.

Reply:
@thomasdax98 I think this will not work, minimal do exist since nodejs v20, when you replace 18 you have to add the minimal prefix to

Reply:
@kaufmo Now it should be okay, I think a6397e680647db6f986bbcaf1ff45136b7b439cf